redis/endpoint.bal

```diff
@@ -82,7 +88,7 @@ public client class Client {
     # + key - Input key to perform NOT
     # + return - Size of the string stored in the destination key or `error` if an error occurs
```

Comment:
The size of the string stored in the destination key, or `error` in case an error occurs